phalcon/mvc/view/engine/volt.zep

```diff
@@ -285,14 +284,14 @@ class Volt extends Engine implements EngineInterface
 	/**
 	 * Checks if a macro is defined and calls it
 	 */
-	public function callMacro(string! name, array arguments)
+	public function callMacro(string! name, array arguments = []) -> var
 	{
 		var macro;
 
 		if !fetch macro, this->_macros[name] {
 			throw new Exception("Macro '" . name . "' does not exist");
 		}
 
-		return call_user_func_array(macro, arguments);
+		return call_user_func(macro, arguments);
```

Comment:
This will break functions https://github.com/phalcon/cphalcon/blob/2.1.x/phalcon/mvc/view/engine/volt/compiler.zep#L555